docs/01-programs/02-candy-machine/00-overview.md

```diff
@@ -0,0 +1,93 @@
+- Protect your launch against bots via configurable bot taxes and gatekeepers like Captchas.
+- Restrict minting to specific NFT/Token holders or to a curated list of wallets.
+- Create multiple minting groups with different sets of rules.
+- Reveal your NFTs after the launch whilst allowing your users to verify that information.
+- And so much more!
+
+Interested? Let’s give you a little tour of how Candy Machines work!
+
+## The Lifecycle of a Candy Machine
+
+Much like its name suggests, you can think of a Candy Machine as a temporary structure which is first loaded by creators and then unloaded by buyers.
+
+Thus, the very first step is for the creator to create a new Candy Machine and configure it however they want.
+
+![Candy Machines V3 - Overview 1@2x.png](/assets/candy-machine-v3/CandyMachinesV3-Overview1.png#radius)
+
+The created Candy Machine keeps track its own settings which helps us understand how all of its NFTs should be minted. For instance, there is a `creators` parameter which will be assign to all NFTs minted from this Candy Machine. We will see how to create and configure Candy Machines in more details, including some code examples, in the following pages: [Candy Machine Settings](/programs/candy-machine/candy-machine-settings) and [Managing Candy Machines](/programs/candy-machine/managing-candy-machines).
+
+However, we still don’t know which NFTs should be minted from that Candy Machine. In other words, the Candy Machine is not loaded. So our next step, is to insert items into the Candy Machine.
+
+![Candy Machines V3 - Overview 2@2x.png](/assets/candy-machine-v3/CandyMachinesV3-Overview2.png#radius)
+
+Each item is composed of two parameters:
+
+- A `name`: The name of the NFT.
+- A `uri`: The URI pointing to the [JSON metadata](https://docs.metaplex.com/programs/token-metadata/overview#a-json-standard) of the NFT. This implies that the JSON metadata has already been uploaded via either an on-chain (e.g. Arweave, IPFS) or off-chain (e.g. AWS, your own server) storage provider.
+
+All other parameters are shared between all NFTs and are therefore kept in the settings of the Candy Machine directly to avoid repetition. See [Inserting Items](/programs/candy-machine/inserting-items) for more details.
+
+Notice how, at this point, no real NFTs have been created yet. We are simply loading the Candy Machine with all the data it needs to **create NFTs on-demand**, at mint time. Which brings us to the next step.
+
+![Candy Machines V3 - Overview 3@2x.png](/assets/candy-machine-v3/CandyMachinesV3-Overview3.png#radius)
+
+Once the Candy Machine is loaded and all pre-configured conditions are met, users can start minting NFTs from it. It’s only at this point that an NFT is created on the Solana blockchain. Note that, before minting, some users may need to perform additional verification steps — such as doing a Captcha or sending a Merkle Proof. See [Minting](/programs/candy-machine/minting) for more details.
```

Comment:
Users here would be collectors but I didn't want to coin that term here because it as a very strong connotation of "wales" and people that mint all the time. Also technically, creators can mint other creators' NFTs so there are not mutually exclusive. I went for users to stay a bit more generic since anyone can mint.